src/objects/GS_Admin_Checklist_Item__mdt.object

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>NPSP Getting Started Page section items</description>
+    <fields>
+        <fullName>Description_Label__c</fullName>
+        <description>The custom label api name used in the description on a section item on the Getting Started page</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The custom label api name for the Description Label</inlineHelpText>
+        <label>Description Label</label>
+        <length>255</length>
+        <required>true</required>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Extra_Info_Label__c</fullName>
+        <description>The custom label api name used in the extra info on a section item on the Getting Started page</description>
```

Comment:
@diego-caballero @fparodi-salesforce @force2b 

Is this an accurate description of this field and what it's used for? If so, please replace the Description and InlineHelpText with this value: "Provides extra information about a section item (such as video length, PDF, etc). Shown in the upper right corner."